### ISSUE TITLE
Is it difficult to make opening import file of imported file?

### ISSUE THREAD

**Issue by nodkz** [AsaAyers/js-hyperclick#45]
Is it difficult in realization, when we click by `Label` open `app/forms/Label/index.js` instead of `app/forms/index.js`? 

**app/page.js**
```js
import { Button, Label } from 'app/forms';

export default function Component(props) {
  return (
    <div>
      <Label>The button:</Label>
      <Button>Click me</Button>
    </div>
  );
}
```

**app/forms/index.js**
```js
export Label from './Label';
export Button from './Button';
...
```

**app/forms/Label/index.js**
```js
export default class Label extends React.Component {
  ...
}
```

Love this plugin! And want love it more! 😊
Thanks!

**Comment by AsaAyers** [comment]
Sorry for the delay, I totally missed the email about this..

I'm sure this is possible. There is already code in place that finds where the target variable is exported. It just needs to be extended to follow it when it's an `export ... from`. I'm not sure if that should become the default behavior or not though. I wonder if there are any good use cases for not following an `export ... from`.

**Comment by nodkz** [comment]
If user clicked by `filepath` then better to ope file by this path. If clicked by variable name, then better to open definition, rather than show intermediate `export ... from`.

**Comment by AsaAyers** [comment]
I have this working in a local branch, but I'm not sure what to call the setting. It works by checking wherever you landed to see if it's a link it could follow. If it is, then it will follow it and repeat the process. To prevent infinite loops it has a limit on how many automatic jumps in a row it can take.

**Comment by AsaAyers** [comment]
This is available in 1.10.0. This seems useful enough to be on by default. For anyone who doesn't like it, there is a setting.

![selection_036](https://cloud.githubusercontent.com/assets/324999/21468621/90aaca4a-c9d0-11e6-9bd7-1786145daa65.png)

**Comment by nodkz** [comment]
Amazing! Works great! Thanks a lot!!!
